public/javascripts/show_lan_devices_actions.js

```diff
@@ -387,126 +389,117 @@ anlixDocumentReady.add(function() {
       grantLanDevicesBlock = role.grantLanDevicesBlock;
     }
 
-    $('#lan-devices-placeholder').hide();
-    let lanDevsRow = $('#lan-devices-body');
-    let countAddedDevs = 0;
-    let lanRoutersRow = $('#lan-routers-body');
-    let countAddedRouters = 0;
-
-    $.each(lanDevices, function(idx, device) {
-      let isWired = (device.conn_type == 0);
-      let cantBlockWired = !lanDevicesGrantBlockWiredDevices;
-      let deviceDoesNotHavePermission = !lanDevicesGrantBlockDevices;
-      let userDoesNotHavePermission = !(isSuperuser || grantLanDevicesBlock);
-      let cantBlockDevice = (
-        isBridge ||
-        (isWired && cantBlockWired) ||
-        userDoesNotHavePermission ||
-        deviceDoesNotHavePermission
-      );
-
-      // Skip if offline for too long only if not blocked
-      if (device.is_old && !device.is_blocked) {
-        return true;
-      }
+    let isWired = (device.conn_type == 0);
+    let cantBlockWired = !lanDevicesGrantBlockWiredDevices;
+    let deviceDoesNotHavePermission = !lanDevicesGrantBlockDevices;
+    let userDoesNotHavePermission = !(isSuperuser || grantLanDevicesBlock);
+    let cantBlockDevice = (
+      isBridge ||
+      (isWired && cantBlockWired) ||
+      userDoesNotHavePermission ||
+      deviceDoesNotHavePermission
+    );
+
+    // Skip if offline for too long only if not blocked
+    if (device.is_old && !device.is_blocked) {
+      return true;
+    }
```

Comment:
Esse `true` provavelmente está gerando os true's na interface. Validar nos pontos que usam essa função se está vindo em branco para evitar de influenciar no cálculo da quantidade de dispositivos
```suggestion
    if (device.is_old && !device.is_blocked) {
      return '';
    }
```